Automattic/pocket-casts-ios
# New grid tweaks

Fixes #1799

Tweak grid spacing and badge fonts for better readability and fix a scrolling bug.

| Before | After |
| - | - |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-06-12 at 15 15 53](https://github.com/Automattic/pocket-casts-ios/assets/651601/70ff9a02-79fc-4dd2-9647-ce5864184d62) | ![Simulator Screenshot - iPhone 15 Pro - 2024-06-12 at 14 54 28](https://github.com/Automattic/pocket-casts-ios/assets/651601/94bd9dfd-d868-4fe9-9a7f-3f26408da7a8) |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-06-12 at 15 15 44](https://github.com/Automattic/pocket-casts-ios/assets/651601/42966629-5427-45fd-92d9-9c097e95e91a) | ![Simulator Screenshot - iPhone 15 Pro - 2024-06-12 at 14 54 42](https://github.com/Automattic/pocket-casts-ios/assets/651601/1729a3a3-3a04-457f-ba37-bad8675675e6)  |






## To test

- Start the app
- Open the podcasts tab
- Turn on the badges for unfinished episodes
- Turn on the grid view
- Check if the layout is correct
- Scroll down  the podcasts list
- Tap on the podcasts tab icon on the tab bar
- The view should scroll up and keep the edges correct
- Repeat the test with the other size of grid view and the list view

## Checklist
